shared/common-adapters/virtualized-list.native.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import * as React from 'react'
+import {VirtualizedList} from 'react-native'
+
+class NativeVirtualizedList extends React.Component<React.ElementProps<VirtualizedList>> {
+  _mounted = true
```

Comment:
Is it safe to make this `true` before `componentDidMount`? Seems like it could replace one cause of yellow boxes with another.

Reply:
in practice yes since the callbacks never happen before componentDidMount, essentially if the component is alive its ok, if its on its way to being dead then its not, so we can safely init it here. Maybe it should be called _safeToCallSetState or something but we do this everywhere

Reply:
we do this pretty inconsistently: https://github.com/keybase/client/search?p=1&q=_mounted&unscoped_q=_mounted

Reply:
let me just change them all